contents/founders/learnings-from-elon.md

```diff
@@ -1,48 +1,47 @@
 
 ## The power (or not) of artificial deadlines and urgency
 
-There are many examples of Elon setting a ridiculous seeming artificial deadline to create urgency. For example, with the Tesla model 3 - in 2016, Elon announced it would advance its 500,000 unit plan to 2018, two years earlier than previously planned.
+There are many examples of Elon setting a ridiculous seeming artificial deadline to create urgency. In 2016 with the Tesla Model 3, for example, Elon announced it would advance its 500,000 unit plan to 2018, two years earlier than previously planned.
 
-This clearly works for him - look at all the progress he has made. Getting to space, totally disrupting the automative industry world wide, and so on!
+This clearly works for him – look at all the progress he has made. Getting to space, totally disrupting the automative industry worldwide, and so on!
 
-There are other companies in our space doing the same - Supabase, for example, are known for their [launch weeks](https://supabase.com/blog/supabase-how-we-launch).
+There are other companies in our space doing the same. Supabase, for example, is known for its [launch weeks](https://supabase.com/blog/supabase-how-we-launch).
 
 At PostHog, artificial deadlines have not worked. We even tried doing a launch week once. We’ve considered why this is the case quite carefully. We think it comes down to a particularly intrinsically motivated team. Let me explain.
 
 Part of our strategy is to ship all the tools developers need. This means we’ve a lot of surface area - too much for one person or team to hold it all in their heads. We need to decentralize control of our products to our small teams. This means each team decide what to build proactively. So we hire proactive, intrinsically motivated people.
 
-Second, we think more experienced people are usually better at working proactively - since they get stuck less often and need less help. These people generally do not want to work until midnight on a weekend, as they usually, but not always, have families because they’re normally older. We’ve gone so far as to have an all remote environment, which is particularly well suited to people like this (amongst many other benefits, such as long, uninterrupted blocks of time to just build cool stuff).
+Second, we think more experienced people are usually better at working proactively, since they get stuck less often and need less help. These people generally do not want to work until midnight on a weekend, as they usually, but not always, have families because they’re normally older. We’ve gone so far as to have an all remote environment, which is particularly well suited to people like this (amongst many other benefits, such as long, uninterrupted blocks of time to just build cool stuff).
```

Comment:
```suggestion
Second, we think more experienced people are usually better at working proactively, since they get stuck less often and need less help. These people generally do not want to work until midnight on a weekend as they usually, but not always, have families. We’ve gone so far as to have an all remote environment, which is particularly well suited to people like this (amongst many other benefits, such as long, uninterrupted blocks of time to just build cool stuff).
```